Kickstarter-iOS/Views/Storyboards/PaymentMethodsFooterView.xib

```diff
@@ -16,54 +16,45 @@
             <rect key="frame" x="0.0" y="0.0" width="375" height="81"/>
             <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
             <subviews>
-                <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="uK7-rQ-yzD">
-                    <rect key="frame" x="0.0" y="0.0" width="375" height="30"/>
-                    <color key="backgroundColor" red="0.96862745100000003" green="0.96862745100000003" blue="0.96470588239999999" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
-                    <constraints>
-                        <constraint firstAttribute="height" constant="30" id="nDJ-JW-rci"/>
-                    </constraints>
-                </view>
-                <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="rxR-Rr-zSv">
-                    <rect key="frame" x="0.0" y="37" width="375" height="44"/>
+                <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="QRi-RW-gGm" userLabel="Content View">
+                    <rect key="frame" x="0.0" y="0.0" width="375" height="81"/>
                     <subviews>
                         <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="left" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="ZVF-Hv-L8e">
-                            <rect key="frame" x="0.0" y="0.0" width="375" height="44"/>
+                            <rect key="frame" x="0.0" y="0.0" width="375" height="81"/>
                             <constraints>
-                                <constraint firstAttribute="height" constant="44" id="RY1-Id-ADx"/>
+                                <constraint firstAttribute="height" relation="greaterThanOrEqual" priority="750" constant="44" id="RY1-Id-ADx"/>
```

Comment:
Lowered the priority of the height constraint so there's no ambiguity with the constraints the table view gives this view.